src/bio/bio_device.c

```diff
@@ -1012,21 +1173,33 @@ bio_led_manage(struct bio_xs_context *xs_ctxt, char *tr_addr, uuid_t dev_uuid, u
 	       unsigned int *state, uint64_t duration)
 {
 	struct spdk_pci_addr	pci_addr;
+	int                     addr_len = 0;
 	int			rc;
 
+	/* LED management on NVMe devices currently only supported when VMD is enabled. */
+	if (!bio_vmd_enabled)
+		return 0;
```

Comment:
[Question]: should we return -DER_NOTSUPPORTED here?

Reply:
we could but then we need to catch it later and convert to successful result so we can run on non-VMD enabled platforms, so I think returning success is okay